internal/events/sql.go

```diff
@@ -30,6 +30,7 @@ const (
 	insertEventIdempotent  = `INSERT INTO ` + SchemaName + `.events (id, data, event_type) VALUES ($1, $2, $3) ON CONFLICT DO NOTHING;`
 	deleteEvent            = `DELETE FROM ` + SchemaName + `.events WHERE id = $1;`
 	getEvents              = `SELECT data, event_type FROM ` + SchemaName + `.events;`
+	getEventsLimit         = `SELECT data, event_type FROM ` + SchemaName + `.events LIMIT $1;`
```

Comment:
Is this used somewhere? probably include the #events in the GetEvents(ctx, db, #events)

Reply:
Ah I made something for it but got rid of it